test/e2e/flask/multichain-api/get-session.spec.ts

```diff
@@ -0,0 +1,146 @@
+import _ from 'lodash';
+import * as path from 'path';
+import {
+  DAPP_URL,
+  largeDelayMs,
+  openDapp,
+  unlockWallet,
+  WINDOW_TITLES,
+  withFixtures,
+} from '../../helpers';
+import { Driver } from '../../webdriver/driver';
+
+import FixtureBuilder from '../../fixture-builder';
+
+const OPTIONS = {
+  dapp: true,
+  dappPaths: [
+    path.join(
+      '..',
+      '..',
+      'node_modules',
+      '@metamask',
+      'test-dapp-multichain',
+      'build',
+    ),
+  ],
+  fixtures: new FixtureBuilder().withPopularNetworks().build(),
+};
+
+async function openDappAndConnectWallet(
+  driver: Driver,
+  extensionId: string,
+): Promise<void> {
+  await unlockWallet(driver);
+
+  await openDapp(driver, undefined, DAPP_URL);
+
+  await driver.fill('[placeholder="Enter extension ID"]', extensionId);
+  await driver.clickElement({ text: 'Connect', tag: 'button' });
+  await driver.delay(largeDelayMs);
+}
+
+describe('Multichain API', function () {
+  describe('Connect wallet to the multichain dapp via `externally_connectable`, call `wallet_getSession` with no session created:', function () {
+    it('should successfully receive empty session scopes', async function () {
+      await withFixtures(
+        {
+          title: this.test?.fullTitle(),
+          ...OPTIONS,
+        },
+        async ({
+          driver,
+          extensionId,
+        }: {
+          driver: Driver;
+          extensionId: string;
+        }) => {
+          await openDappAndConnectWallet(driver, extensionId);
+
+          await driver.clickElement('#get-session-btn');
+
+          const notificationList = await driver.findElements(
+            '.notification-container',
+          );
+          const sessionResultSummary = notificationList[0];
+          await sessionResultSummary.click();
+
+          const completeRawResult = await driver.findElement(
+            '#session-method-result-0',
+          );
+          const parsedResult = JSON.parse(await completeRawResult.getText());
+
+          if (!_.isEmpty(parsedResult.sessionScopes)) {
+            throw new Error('Should receive empty session scopes');
+          }
+        },
+      );
+    });
+  });
+
+  describe('Connect wallet to the multichain dapp via `externally_connectable`, call `wallet_getSession` with multiple sessions created:', function () {
```

Comment:
```suggestion
  describe('Connect wallet to the multichain dapp via `externally_connectable`, call `wallet_getSession` when there is an existing session', function () {
```

Reply:
Resolved in [a105b35](https://github.com/MetaMask/metamask-extension/pull/29298/commits/a105b35b9d82ff0657fb79f9a42d29760b21e09a)